spec.bs

```diff
@@ -647,6 +647,33 @@ Each time a <{fencedframe}> navigates to a [=fenced frame config=], it is instan
 [=fenced frame config instance=], which governs the particular [=browsing context group=] inside the
 [=fenced navigable container/fenced navigable=].
 
+<h4 id=fenced-frame-config-use-cases>Use cases</h4>
+
+Rendering an ad created through an ad auction:
+
+An ad auction API runs an auction and determines a winning ad. Details about the winning ad must be
+hidden from the embedder, and the embedding context cannot be allowed to influence the environment
```

Comment:
```suggestion
hidden from the embedder, and the embedding context is not allowed to influence the environment
```

..just to avoid normative language even in this section (i.e., https://infra.spec.whatwg.org/#conformance)